SIT/environment.sh

```diff
@@ -69,7 +69,7 @@ fi
 # These variables are hard-coded for now
 SUBNET=$(echo ${CIDR} | cut -d. -f1-3)
 CASS_VERSION=4
-CDM_VERSION=feature-spark_scala_scc
+CDM_VERSION=main
```

Comment:
This will be later updated to `latest` docker image version when we cut the new release.